src/Templates/TemplateSolutions/CustomTypeEditor/CustomControlLibrary.Design.Server/CustomControlLibrary.Design.Server.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>net6.0-windows</TargetFramework>
+		<UseWindowsForms>true</UseWindowsForms>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.WinForms.Designer.SDK" Version="1.1.0-prerelease-preview3.22076.5" />
+	</ItemGroup>
```

Comment:
This entry is repeated over and over, and becomes a maintenance issue. There are few ways this can be tackled.
1. Use [Central Package Management](https://docs.microsoft.com/nuget/consume-packages/central-package-management#enabling-central-package-management)
2. A manual version of the CPM - i.e., have a file with reference versions and then use MSBuild variables. This is similar to how we managed package versions in our repos (e.g., https://github.com/dotnet/winforms/blob/main/eng/Versions.props).

Reply:
There is one other issue, I need to discuss with you in this context.
I'd like a meeting where we can through this together, and come up with a suggestion for this?
It's too complex a scenario to describe it here; I'd need to show you.

Reply:
Feel free to ping me when you're ready